pylint_odoo/checkers/no_modules.py

```diff
@@ -232,6 +240,20 @@ def visit_callfunc(self, node):
                     self.add_message('translation-field',
                                      node=argument_aux)
 
+        # Check if cr.execute() has an sql-injection
+        if node.as_string().lower().startswith('cr.execute(') or \
+                "cr.execute(" in node.as_string():
+            if len(node.args) is 1:
+                arg_string = node.args[0].as_string()
```

Comment:
You can get all tokens from a node to get the exactly `%` case.
Could you see other original cases from pylint or check other methods allowed from a pdb?
To avoid use a node.as_string() and avoid split manually the tokens with a regex.